intro-HPC/ch_introduction.tex

```diff
@@ -285,11 +340,19 @@ \subsection{What programming languages can I use?}
 M(VA)PICH, OpenMPI.
 \fi
 \ifleuven
-Commonly used software packages are ABINIT, CP2K, Gaussian, Gromacs, Molpro,
-NWChem, Quantum Espresso, R, Siesta, TURBOMOLE, WIEN2k, \ldots
-
-Commonly used Libraries are Intel MKL, FFTW, HDF5, PETSc and Intel MPI,
-M(VA)PICH, OpenMPI.
+Commonly used software packages are:
+\begin{itemize}
+\item{in bioinformatics: beagle, Beast, bedtools, bowtie, BWA, Mr. Bayes, TopHat, TRIQS,}
```

Comment:
I do not see installed in Leuven

From: Kenneth Hoste [mailto:notifications@github.com]
Sent: donderdag 18 februari 2016 14:06
To: hpcugent/vsc_user_docs <vsc_user_docs@noreply.github.com>
Cc: Mag Selwa <mag.selwa@kuleuven.be>
Subject: Re: [vsc_user_docs] Feature/leuven chapter1 (#106)


In intro-HPC/ch_introduction.tex<https://github.com/hpcugent/vsc_user_docs/pull/106#discussion_r53310467>:

> @@ -285,11 +340,19 @@ \subsection{What programming languages can I use?}

>  M(VA)PICH, OpenMPI.

>  \fi

>  \ifleuven

> -Commonly used software packages are ABINIT, CP2K, Gaussian, Gromacs, Molpro,

> -NWChem, Quantum Espresso, R, Siesta, TURBOMOLE, WIEN2k, \ldots

> -

> -Commonly used Libraries are Intel MKL, FFTW, HDF5, PETSc and Intel MPI,

> -M(VA)PICH, OpenMPI.

> +Commonly used software packages are:

> +\begin{itemize}

> +\item{in bioinformatics: beagle, Beast, bedtools, bowtie, BWA, Mr. Bayes, TopHat, TRIQS,}

BLAST

—
Reply to this email directly or view it on GitHub<https://github.com/hpcugent/vsc_user_docs/pull/106/files#r53310467>.